Anlab.Mvc/Controllers/LabController.cs

```diff
@@ -119,9 +119,23 @@ public IActionResult ListUsersOrders(string id)
                 ErrorMessage = "A request number is required";
                 return RedirectToAction("Confirmation");
             }
+            order.RequestNum = requestNum;
+            var result = await _orderService.OverwiteOrderFromDb(order); //TODO: Just testing

```

Comment:
@laholstege this will probably have to change since there could be other errors, like work request number not being found.  But for now it's fine, let's just remember to come back here after the SQL DB is back up